.devcontainer/devcontainer.json

```diff
@@ -8,17 +8,23 @@
   // Features to add to the dev container. More info: https://containers.dev/features.
   "features": {
     "ghcr.io/devcontainers-contrib/features/pnpm": {
-      "version": "9.1.3"
+      "version": "9.8.0"
     },
-    "ghcr.io/joshuanianji/devcontainer-features/mount-pnpm-store": {}
+    "ghcr.io/joshuanianji/devcontainer-features/mount-pnpm-store": {},
+    "ghcr.io/devcontainers-contrib/features/npm-package": {
+      "package": "radashi-helper",
+      "version": "latest"
+    }
   },
-  "postCreateCommand": "pnpm i",
+  "postStartCommand": "pnpm i",
```

Comment:
What's the difference between postCreate and postStart here?

Reply:
Both commands are similar, but `postCreateCommand` runs during the container's initial setup, while `postStartCommand` is only executed once the container is fully functional. For me, it makes more sense to install dependencies in `postStartCommand`, ensuring the container is ready. In this current scenario, it doesn't make a difference, but I updated it to make it more semantic.

https://containers.dev/implementors/json_reference/#lifecycle-scripts